### PR TITLE
Allow integer as sequence_lenghts in compute_masks

### PR DESCRIPTION
**Pull request by jensheit**
adds possiblity to use integers as sequence_lengths in compute_masks

### CODE REVIEW

File: padertorch/ops/sequence/mask.py [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import torch
+from typing import Union, List
 
 
-def compute_mask(x, sequence_lengths, batch_axis=0, sequence_axis=1):
+def compute_mask(x: torch.Tensor,
+                 sequence_lengths: Union[List[int], int, None],
+                 batch_axis: int = 0, sequence_axis: int = 1):
```
Comment (boeddeker) [inline padertorch/ops/sequence/mask.py:7]: Can you add a line break between `batch_axis` and `sequence_axis`?
```python
def compute_mask(
        x: torch.Tensor,
        sequence_lengths: Union[List[int], int, None],
        batch_axis: int = 0,
        sequence_axis: int = 1,
):
```

**Comment by boeddeker** [comment]
What is the motivation of `sequence_lenghts: int`?
I expected that `sequence_lenghts is None` uses the faster/better code path, while yielding the same result.

**Comment by jensheit** [comment]
> What is the motivation of `sequence_lenghts: int`?
> I expected that `sequence_lenghts is None` uses the faster/better code path, while yielding the same result.

In theory there might be some array I have padded for convolution or other reasons, where I all original signals have the same length.
